chore(ts): Suffix all fixtures with `Fixture`

gggritso opened the pull request:
This disambiguates them from types.

https://github.com/getsentry/frontend-tsc/issues/49

gggritso committed:
Suffix all fixtures with `Fixture`

gggritso committed:
Unroll change from another branch

gggritso committed:
Merge branch 'master' into chore/ts/explicit-exports

gggritso committed:
Merge branch 'master' into chore/ts/explicit-exports

gggritso committed:
Fix incorrect fixture load

gggritso committed:
Fix new incorrect imports

JonasBa commented:
Very welcome change as naming conflicts end up in inconsistent aliasing that we can entirely avoid with a change like this.

gggritso committed:
Merge branch 'master' into chore/ts/explicit-exports

ryan953 commented:
There was a previous discussion that this aligns with: https://www.notion.so/sentry/68089bf1ccdd480487a6ec4447cbd667

The consensus at the time was that: naming collisions are bad and we want to disambiguate things. Some options were given but the result was:
```diff
// Descision:
  import {Organization} from 'sentry/types';
- import {Organization as OrganizationFixture} from 'sentry/fixtures/organization';
  // later we will rename the file and endup with:
+ import {OrganizationFixture} from 'sentry/fixtures/organizationFixture';
```

gggritso committed:
Add export aliases

`getsentry` needs these to run the suite. When the rename is completely,
I can remove these.
